forge-gui/res/cardsfolder/g/glen_elendra.txt

```diff
@@ -2,7 +2,7 @@ Name:Glen Elendra
 ManaCost:no cost
 Types:Plane Lorwyn
 T:Mode$ Phase | Phase$ EndCombat | ValidPlayer$ You | TriggerZones$ Command | OptionalDecider$ You | Execute$ TrigExchange | TriggerDescription$ At end of combat, you may exchange control of target creature you control that dealt combat damage to a player this combat and target creature that player controls.
-SVar:TrigExchange:DB$ Pump | ValidTgts$ Creature.YouCtrl+dealtCombatDamageThisCombat Player | TgtPrompt$ Select target creature you control that dealt combat damage to a player | SubAbility$ DBExchange
+SVar:TrigExchange:DB$ Pump | ValidTgts$ Creature.YouCtrl+dealtCombatDamageThisCombat | TgtPrompt$ Select target creature you control that dealt combat damage to a player | SubAbility$ DBExchange
```

Comment:
What if the creature dealt combat damage to a creature or planeswalker? Is Player a default or something?

Reply:
Yea since this is the only card like that I changed it so it doesn‘t track any unneeded stuff